### PR TITLE
Update install.sh for plugins/arcthemesolid.plugin/ for Fedora 24

### PR DESCRIPTION
**Pull request by jvb187**
Update install.sh for plugins/arcthemesolid.plugin/ for Fedora 24

Repo download path changed for Fedora 24:
http://download.opensuse.org/repositories/home:Horst3180/Fedora_24_standard/home:Horst3180.repo

### CODE REVIEW

File: plugins/arcthemesolid.plugin/install.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
-dnf config-manager --add-repo=http://download.opensuse.org/repositories/home:Horst3180/Fedora_$(cat /etc/fedora-release | grep -o "[0-9]*")/home:Horst3180.repo
+dnf config-manager --add-repo=http://download.opensuse.org/repositories/home:Horst3180/Fedora_$(cat /etc/fedora-release | grep -o "[0-9]*")_standard/home:Horst3180.repo
```
Comment (satya164) [inline plugins/arcthemesolid.plugin/install.sh]: Does this work on older Fedora versions? If not can we a add a check?

Comment (satya164) [inline plugins/arcthemesolid.plugin/install.sh]: Does this work on older Fedora versions? If not can we a add a check?

**Comment by jvb187** [comment]
I added a simple if then to check for 24

File: plugins/arcthemesolid.plugin/install.sh [hunk 2]
```diff
@@ -1,4 +1,8 @@
 #!/bin/bash
 
-dnf config-manager --add-repo=http://download.opensuse.org/repositories/home:Horst3180/Fedora_$(cat /etc/fedora-release | grep -o "[0-9]*")/home:Horst3180.repo
+if [[ $(cat /etc/fedora-release | grep -o "[0-9]*") = 24 ]]; then
```
Comment (satya164) [inline plugins/arcthemesolid.plugin/install.sh]: can we do a greater than equals check instead of just equals?

Comment (jvb187) [inline plugins/arcthemesolid.plugin/install.sh]: Updated to greater than or equals to

**Pull request merged**